bin/hbase-config.sh

```diff
@@ -168,3 +168,27 @@ if [ -z "$JAVA_HOME" ]; then
 EOF
     exit 1
 fi
+
+function read_java_version() {
+  properties="$("${JAVA_HOME}/bin/java" -XshowSettings:properties -version 2>&1)"
```

Comment:
This in jdk8 too?

Reply:
It's there for everything I have installed. Granted they're all openjdk variants, and mostly adopt-openjdk packages.

```
$ JAVA_HOME=/Library/Java/JavaVirtualMachines/zulu-7.jdk/Contents/Home java -XshowSettings:properties -version 2>&1 | grep java.runtime.version
    java.runtime.version = 1.7.0_232-b6
$ JAVA_HOME=/Library/Java/JavaVirtualMachines/adoptopenjdk-8.jdk/Contents/Home java -XshowSettings:properties -version 2>&1 | grep java.runtime.version
    java.runtime.version = 1.8.0_222-b10
$ JAVA_HOME=/Library/Java/JavaVirtualMachines/adoptopenjdk-9.jdk/Contents/Home java -XshowSettings:properties -version 2>&1 | grep java.runtime.version
    java.runtime.version = 9+181
$ JAVA_HOME=/Library/Java/JavaVirtualMachines/adoptopenjdk-10.jdk/Contents/Home java -XshowSettings:properties -version 2>&1 | grep java.runtime.version
    java.runtime.version = 10.0.2+13
$ JAVA_HOME=/Library/Java/JavaVirtualMachines/adoptopenjdk-11.jdk/Contents/Home java -XshowSettings:properties -version 2>&1 | grep java.runtime.version
    java.runtime.version = 11.0.4+11
$ JAVA_HOME=/Library/Java/JavaVirtualMachines/adoptopenjdk-12.jdk/Contents/Home java -XshowSettings:properties -version 2>&1 | grep java.runtime.version
    java.runtime.version = 12.0.2+10
$ JAVA_HOME=/usr/local/Cellar/openjdk/13.0.2+8_2 java -XshowSettings:properties -version 2>&1 | grep java.runtime.version
    java.runtime.version = 13.0.2+8
```